app/Console/Commands/RebuildImages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
```

Comment:
Most of Pony.fm's source files, including all of its PHP files, should include the AGPL license header. The easy way to add one is to copy-paste it from another PHP file, change the copyright attribution to your name, and set the year to the current one.